qa/workunits/rbd/cli_generic.sh

```diff
@@ -1336,6 +1346,12 @@ test_mirror_snapshot_schedule() {
     test "$(rbd mirror image status rbd2/ns1/test1 |
         grep -c mirror.primary)" = '1'
 
+    # check that remove fails when image exists but no schedules are in place
+    expect_fail rbd mirror snapshot schedule remove dummy
+    expect_fail rbd mirror snapshot schedule remove 1h dummy
+    expect_fail rbd mirror snapshot schedule remove -p rbd2/ns1 --image test1 dummy
```

Comment:
-p with `<pool>/<namespace>` It's wrong way of specifying Is it? Or we are validating this also in negative scenario here
can we also please cover `-p <poolname> --namespace <name> `commands as well?